Gordon360/ApiControllers/WellnessController.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Linq;
+using System.Security.Claims;
+using System.Web.Http;
+using System.Net;
+using System.Net.Http;
+using Gordon360.Exceptions.ExceptionFilters;
+using Gordon360.Repositories;
+using Gordon360.Services;
+using Gordon360.Models;
+using Gordon360.Exceptions.CustomExceptions;
+
+namespace Gordon360.Controllers.Api
+{
+    [RoutePrefix("api/wellness")]
+    [CustomExceptionFilter]
+    [Authorize]
+    public class WellnessController : ApiController
+    {
+        private IProfileService _profileService;
+        private IAccountService _accountService;
+        private IRoleCheckingService _roleCheckingService;
+
+        private IWellnessService _wellnessService;
+
+        public WellnessController()
+        {
+            var _unitOfWork = new UnitOfWork();
+            _wellnessService = new WellnessService();
+            _profileService = new ProfileService(_unitOfWork);
+            _accountService = new AccountService(_unitOfWork);
+            _roleCheckingService = new RoleCheckingService(_unitOfWork);
+        }
+
+        public WellnessController(IWellnessService wellnessService)
+        {
+            _wellnessService = wellnessService;
+        }
+
+        /// <summary>
+        ///  Gets current status os student
+        /// </summary>
+        /// <returns>A boolean</returns>
+        [HttpGet]
+        [Route("")]
```

Comment:
the controller creates the main route that we make the Http requests to at the top. The other routes that can be declared there are actually sub routes. You only need to use sub routes to if you have multiple of the same request in your controller. Since I only have a Get and a Post request i don't need to specify a route because the the fact that they are different request differentiates between them. If I had 2 Post than I would have to specify a sub route for at least one of them.